docs/source/about/advanced_usage.rst

```diff
@@ -631,9 +631,29 @@ keybindings
       .. todo:: macOS
 
       **Windows**
-         .. code-block:: batch
+         During Sunshine startup, you should see the list of detected display devices:
 
-            tools\dxgi-info.exe
+         .. code-block:: text
+
+            DEVICE ID: \\?\DISPLAY-ACI27EC-5&4fd2de4&2&UID4355-{e6f07b5f-ee97-4a90-b076-33f57bf4eaa7}
```

Comment:
Changed, just keep in mind that in the end I hash the string and convert it to UID, so it will look like this `{bc172e6d-86eb-5851-aeca-56525ed716e9}`